### PR TITLE
All of the steps for issue 2 have been fixed. User can see their entrie…

### PR DESCRIPTION
**Pull request by BrianBozi**
Users can see their entries and make new ones.
code-journal issue 2

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -23,6 +23,63 @@ $newPost.addEventListener('submit', function (event) {
...
+  var $pElement = document.createElement('p');
+  $pElement.textContent = entry.comment;
+  $divColumnHalf2.append($pElement);
+
+  return $li;
+}
+
+var $ul = document.querySelector('ul');
+
+window.addEventListener('DOMContentLoaded', function (event) {
```
Comment (BrettAlbright) [inline js/main.js]: this should be `document.addEventListener`

File: css/styles.css [hunk 2]
```diff
@@ -4,26 +4,49 @@ body {
...
+}
+
 header > h2 {
   height: 2rem;
 }
 
 img {
   width: 100%;
-  height: ;
+  height: 400px;
```
Comment (BrettAlbright) [inline css/styles.css]: I would remove this `height` property. It is better to just set the width and let the height autogenerate

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
2 small updates!

**Comment by BrianBozi** [comment]
image

**Comment by BrianBozi** [comment]
image

**Review by Zhiwutian — APPROVED** [review]
Approved!!!

**Pull request merged**